src/tests/issues.targets

```diff
@@ -1164,18 +1164,15 @@
 
         <!-- Covariant returns -->
         <!-- https://github.com/dotnet/runtimelab/issues/205 -->
```

Comment:
With this PR we can close https://github.com/dotnet/runtimelab/issues/205. Thanks!

```suggestion
        <!-- Tests that expect TypeLoadException -->
```